firepit/sqlstorage.py

```diff
@@ -959,7 +951,21 @@ def timestamped(
             cursor.close()
         else:
             res = qry
-        return res
+        return res    
+    
+    def timestamped(
+            self,
+            viewname,
+            path=None,
+            value=None,
+            timestamp='first_observed',
+            limit=None,
+            run=True):
+        """
+        Get the timestamped observations of `value` in `viewname`.`path`
+        Returns list of dicts like {'timestamp': '2021-10-...', '{column}': '...'}
+        """
+        return extract_observeddata_attribute(viewname, timestamp)
```

Comment:
You are dropping the `path`, `value`, `limit`, and `run` args.  I guess the unit tests are incomplete here?